index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tekle's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+      integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="container">
+        <div class="logo">
+          <a href="#" class="logo-link">My Logo</a>
+        </div>
+        <nav class="nav-menu">
+          <a href="#" class="btn-menu-toggle"><i class="fas fa-bars"></i></a>
+        </nav>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container">
+        <h1 class="headline--title">I’m Tekle</h1>
+        <h2 class="headline--title subtitle">Glad to see you!</h2>
+        <p class="headline--body">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <p class="headline--supporting-text">Let's connect</p>
+
+        <ul class="social-media">
+          <li class="social-meida__list-item">
+            <a href="" class="social-media-items twitter-link">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li class="social-meida__list-item">
+            <a href="" class="social-media-items linkedin-link">
+              <i class="fab fa-linkedin-in"></i>
+            </a>
+          </li>
+          <li class="social-meida__list-item">
+            <a href="" class="social-media-items medium-link">
+              <i class="fab fa-medium-m"></i>
+            </a>
+          </li>
+          <li class="social-meida__list-item">
+            <a href="" class="social-media-items github-link">
+              <i class="fab fa-github"></i>
+            </a>
+          </li>
+          <li class="social-meida__list-item">
+            <a href="" class="social-media-items angelist-link">
+              <i class="fab fa-angellist"></i>
+            </a>
+          </li>
```

Comment:
- [ ] [OPTIONAL] Also here, I think it could be a good idea to update these links with your socials for a better user experience (UX) :smiley: :raised_hands:

Reply:
thanks. I will.